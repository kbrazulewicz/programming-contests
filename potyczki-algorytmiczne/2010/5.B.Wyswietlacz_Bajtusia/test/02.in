5
1 5 11111
2 2 13
5 20 19283742841042112311
3 20 19283742841042112311
10 20 11111111111111111111
