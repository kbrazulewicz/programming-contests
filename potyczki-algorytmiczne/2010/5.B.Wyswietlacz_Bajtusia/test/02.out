71111
97
99999874442222111111
99998744422221111110
99771111111111111111
